PF/Assignment3/Reminder.java

```diff
@@ -0,0 +1,19 @@
+public class Reminder {
+
+	public static int rem(int x, int y) {
+		if (x >= 0 && y > 0) {
+
+			if (x >= y) {
+				return rem(x - y, y);
+			} else
+				return x;
+		} else {
+			System.out.print("Divide by zero not possible");
+
+			return -1;
```

Comment:
ACTION: You can't return -1 as it can be a valid result. Moreover, if x=-10 and y=2 then also it will print "Divide by zero not possible" which is misleading right? Simply throw an exception in this case.